content/security/terminology.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: security
+title: "Terminology"
+section: security
+---
+
+This page provides a quick overview of commonly used security terms in Jenkins and its documentation.
+While not exhaustive, it aims to clarify terminology used in security advisories and security-related documentation on this site.
+
+## General
+
+Attack::
+As a noun, an attack is a deliberate attempt to _exploit_ a _vulnerability_ to compromise the confidentiality, integrity, or availability of a system.
+As a verb, it means to perform actions intended to compromise, damage, or gain unauthorized access to a system.
+Attacker::
+An attacker is an individual, program, or group that attempts to _exploit_ _vulnerabilities_ in a system.
+Crafted::
+In _vulnerability_ descriptions, "crafted" refers to input or data that has been intentionally designed or manipulated by an _attacker_ to trigger a _vulnerability_.
+Exploit::
+As a noun, an exploit is software, code, or a sequence of instructions that leverages a _vulnerability_ to perform an _attack_.
+As a verb, it means to take advantage of a specific _vulnerability_ in a system.
+Fix::
+A fix is a code or configuration change that removes a _vulnerability_, thereby preventing it from being _exploited_.
+Mitigation::
+A mitigation is a security measure or control that reduces the chances of a _vulnerability_ being _exploited_, or lessens the damage if it is.
+Unlike a _fix_, a mitigation does not necessarily eliminate the _vulnerability_.
+Vulnerability::
+A vulnerability is a flaw or weakness that could be leveraged by an _attacker_ to perform unauthorized actions with a security impact.
+
+## Vulnerabilities
+
+Command Injection::
+Command injection (or OS command injection) is a _vulnerability_ that occurs when an application constructs and executes OS shell commands using unsafe user input.
+Cross-Site Request Forgery (CSRF)::
+CSRF _attacks_ trick a user's browser into performing unwanted actions in a web application they are logged into, often without the user's knowledge.
+link:/security/vulnerabilities/#csrf[Learn more.]
+Cross-Site Scripting (XSS)::
+XSS _vulnerabilities_ allow an _attacker_ to inject malicious scripts into web pages viewed by other users.
+These scripts run within the victim's web browser and can steal sensitive data, hijack sessions, modify content, or perform unauthorized actions on behalf of the user.
+link:/security/vulnerabilities/#xss[Learn more.]
+Disclosure::
+Disclosure refers to a type of _vulnerability_ where sensitive information becomes accessible to unauthorized actors.
+[#pt]
+Path Traversal::
+A path traversal _vulnerability_ allows an _attacker_ to access files and directories that are stored outside the intended directory.
+This is typically achieved by manipulating file path parameters (e.g., using `../` sequences) to traverse the filesystem and access sensitive resources such as configuration files or cryptographic secrets.
```

Comment:
```suggestion
This is typically achieved by manipulating file path parameters, such as using `../` sequences, to traverse the filesystem and access sensitive resources such as configuration files or cryptographic secrets.
```
Suggestion to replace `e.g.` with `such as` per [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).